src/pages/Homepage/HeroSlideshow.vue

```diff
@@ -50,7 +50,32 @@
 				&nbsp;
 			</div>
 		</div>
-		<div class="action-button-wrapper">
+		<div
```

Comment:
@BoulderBrains you shouldn't need to duplicate this markup. You can just drop a span after the copy in each button `Sign up<span v-if="showDoubleArrowButton"> >></span>`